ReSharper.FSharp/test/data/features/generate/overrides/Anchor - Whitespace 12.fs.gold

```diff
@@ -0,0 +1,17 @@
+﻿Provided elements:
+ 0: ToString():System.String
+ 1: Equals(System.Object):System.Boolean
+ 2: GetHashCode():System.Int32
+ 3: Finalize():System.Void
+
+// ${KIND:Overrides}
+// ${SELECT0:ToString():System.String}
+
+type A() =
+    class end
+
+and B() =
+    class end
+
+
+    override this.ToString() = {selstart}failwith "todo"{selend}
```

Comment:
I think we should also update `GenerateOverrides.getAnchorNode`, because it looks like the anchor is outside the type declaration now.